mongo/integration/client_side_encryption_spec_test.go

```diff
@@ -54,6 +55,10 @@ func TestClientSideEncryptionSpec(t *testing.T) {
 
 	for _, fileName := range jsonFilesInDir(t, path.Join(dataPath, encryptionSpecName)) {
 		t.Run(fileName, func(t *testing.T) {
+			re := regexp.MustCompile(`fle2\-Range\-.*\-Correctness`)
+			if re.Match([]byte(fileName)) {
+				t.Skipf("Skipping test on macOS due to slow runtime.")
```

Comment:
```suggestion
				t.Skipf("skipping test on macOS due to slow runtime")
```

optional: AFAIK "Skip" messages cannot be wrapped, unlike errors, so I'm not 100% certain they need to follow the same principles as writing errors -- i.e. no punctuation and not capitalized sentences. But it seems in the Go Driver it's a common practice to enforce those rules.